[2024-09-08T17:19:44.319+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-08T17:19:44.335+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_dag.task1 scheduled__2024-09-06T00:00:00+00:00 [queued]>
[2024-09-08T17:19:44.343+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_dag.task1 scheduled__2024-09-06T00:00:00+00:00 [queued]>
[2024-09-08T17:19:44.344+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 4
[2024-09-08T17:19:44.358+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): task1> on 2024-09-06 00:00:00+00:00
[2024-09-08T17:19:44.369+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4230) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-08T17:19:44.371+0000] {standard_task_runner.py:72} INFO - Started process 4236 to run task
[2024-09-08T17:19:44.369+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'catchup_dag', 'task1', 'scheduled__2024-09-06T00:00:00+00:00', '--job-id', '248', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmpfi_uk3nd']
[2024-09-08T17:19:44.375+0000] {standard_task_runner.py:105} INFO - Job 248: Subtask task1
[2024-09-08T17:19:44.463+0000] {task_command.py:467} INFO - Running <TaskInstance: catchup_dag.task1 scheduled__2024-09-06T00:00:00+00:00 [running]> on host 626f2e28dec2
[2024-09-08T17:19:44.958+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abhay' AIRFLOW_CTX_DAG_ID='catchup_dag' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-06T00:00:00+00:00'
[2024-09-08T17:19:44.965+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-08T17:19:44.993+0000] {logging_mixin.py:190} INFO - hello
[2024-09-08T17:19:44.997+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-08T17:19:45.019+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-08T17:19:45.022+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=catchup_dag, task_id=task1, run_id=scheduled__2024-09-06T00:00:00+00:00, execution_date=20240906T000000, start_date=20240908T171944, end_date=20240908T171945
[2024-09-08T17:19:45.134+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-08T17:19:45.163+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-08T17:19:45.169+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
